packages/hardhat-core/test/internal/hardhat-network/provider/modules/eth/methods/getTransactionByHash.ts

```diff
@@ -329,61 +329,6 @@ describe("Eth module", function () {
           assert.equal(tx.from, "0x8a9d69aa686fa0f9bbdec21294f67d4d9cfb4a3e");
         });
 
-        it("should get an existing transaction from goerli", async function () {
```

Comment:
@fvictorio is this logic tested elsewhere or should we also adopt sepolia here?

Reply:
Done